e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs

```diff
@@ -191,6 +191,6 @@ public void Dispose()
             () => string.Empty);
 
         string IContainerRegistryConfig.ImagesTag => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_IMAGES_TAG,
-            () => throw new Exception("Images tag not provided"));
+            () => "mcr.microsoft.com/iotedge/opc-publisher:latest" );
```

Comment:
Should be just `latest`.

Reply:
It is the whole image when the ContainerServer is not set...

Reply:
No, I it should be only the tag. Here is the image URI generation in `IoTHubPublisherDeployment.cs` where we use the tag:

```csharp
            var version = _context.ContainerRegistryConfig.ImagesTag ?? "latest";
            var image = $"{server}/{ns}iotedge/opc-publisher:{version}";
```